diffpy/pdfmorph/pdfmorph_api.py

```diff
@@ -20,17 +20,29 @@
 
 import matplotlib.pyplot as plt
 
-default_config = dict(scale=None, stretch=None, smear=None,
-                      baselineslope=None, qdamp=None)
+_default_config = dict(scale=None, stretch=None, smear=None,
+                       baselineslope=None, qdamp=None)
 
 # map of operation dict
 # TODO: include morphing on psize
-morph_step_dict = dict(scale=morphs.MorphScale,
-                       stretch=morphs.MorphStretch,
-                       smear=[morphs.MorphXtalPDFtoRDF,
-                              morphs.MorphSmear,
-                              morphs.MorphXtalRDFtoPDF],
-                       qdamp=morphs.MorphResolutionDamping)
+_morph_step_dict = dict(scale=morphs.MorphScale,
+                        stretch=morphs.MorphStretch,
+                        smear=[morphs.MorphXtalPDFtoRDF,
+                               morphs.MorphSmear,
+                               morphs.MorphXtalRDFtoPDF],
+                        qdamp=morphs.MorphResolutionDamping)
+
+
+def morph_default_config():
```

Comment:
Can we rename this to `morph_config(**kw)` and process keyword arguments?
The example for `pdfmorph` could then look as follows:
```python
morph_cfg = morph_config(scale=1.01)
...
```